example.py

```diff
@@ -18,7 +23,7 @@
 
 print(
     posthog.feature_enabled(
-        "person-on-events-enabled",
+        args.flag,  # Use the flag from command line arguments
```

Comment:
fwiw, this only changes the first test flag in this case, the subsequent flags will still use the hard coded values. Seems fine if you're only really using the example script for the first case.

Reply:
I’ll cross those bridges if the need arises. 😂